test/fixtures/settings.yml

```diff
@@ -1,402 +1,4 @@
-  name: login_text
-  category: Setting
-  default: "--- \n"
-  description: 'Text to be shown in the login-page footer'
-attribute83:
-  name: ssh_timeout
-  category: Setting
-  default: 120
-  description: 'Time in seconds before SSH provisioning times out'
-attribute84:
-  name: destroy_vm_on_host_delete
-  category: Setting
-  default: true
-  description: 'Destroy associated VM on host delete. When enabled, VMs linked to Hosts will be deleted on Compute Resource, meaning they can be re-associated or imported back to Foreman again. This does not automatically power off the VM'
-attribute85:
-  name: intermediate_ipxe_script
-  category: Setting
-  default: nil
-  description: 'Intermidiate iPXE script'
-attribute86:
-  name: maximum_structured_facts
-  category: Setting
-  default: 100
-  description: 'Maximum structured facts'
-attribute87:
-  name: instance_id
-  category: Setting
-  default: '2abbbe02-4ace-4269-9e20-2753f3206cc2'
-  description: 'Foreman UUID'
-attribute88:
-  name: oidc_jwks_url
-  category: Setting
-  default: 'https://keycloak.example.com/auth/realms/foreman/protocol/openid-connect/certs'
-  description: 'OpenID Connect JSON Web Key Set(JWKS) URL. Typically https://keycloak.example.com/auth/realms/<realm name>/protocol/openid-connect/certs if you are using Keycloak as an IDP'
-attribute89:
-  name: oidc_audience
-  category: Setting
-  default: 'rest-client'
-  description: 'Name of the OpenID Connect Audience that is being used for Authentication. For example in case of Keycloak this is the Client ID.'
-attribute90:
-  name: oidc_issuer
-  category: Setting
-  default: 127.0.0.1
-  description: "The iss (issuer) claim identifies the principal that issued the JWT, which exists at a `/.well-known/openid-configuration` in case of most of the IDP's."
-attribute91:
-  name: oidc_algorithm
-  category: Setting
-  default: 'RS512'
-  description: 'The algorithm used to encode the JWT in the IDP.'
-attribute92:
-  name: authorize_login_delegation_auth_source_user_autocreate
-  category: Setting
-  default: 'External'
-  description: 'Name of the external auth source where unknown externally authentication users (see authorize_login_delegation) should be created (keep unset to prevent the autocreation)'
-attribute93:
-  name: update_hostgroup_from_facts
-  category: Setting
-  default: true
-  description: "Foreman will update a host's hostgroup from its facts"
-attribute94:
-  name: default_global_registration_item
-  category: Setting
-  default: 'Global Registration'
-  description: "Default Global registration template"
-attribute95:
-  name: default_host_init_config_template
-  category: Setting
-  default: 'Linux host initial configuration'
-  description: "Default host initial configuration template"
-attribute96:
-  name: server_ca_file
-  category: Setting
-  default: /var/lib/puppet/ssl/certs/ca.pem
-  description: SSL CA file that will be used in templates (to verify the connection to Foreman)
 attribute97:
```

Comment:
Are these here because the override the default?

Reply:
yeah, we need a different value then a default. I'm not happy about it, I do not agree with a bin path being in settings, but I do not have better solution for it.